modules/javafx.graphics/src/main/java/com/sun/javafx/iio/ImageStorage.java

```diff
@@ -380,7 +380,20 @@ public ImageFrame[] loadAll(String input, ImageLoadListener listener,
                     }
 
                     if (theStream == null) {
-                        theStream = ImageTools.createInputStream(input);
+                        try {
+                            theStream = ImageTools.createInputStream(input);
+                        } catch (IOException ignored) {
+                        }
+                    }
+
+                    if (theStream == null) {
+                        try {
+                            // last fallback, try to see if the file exists with @1x suffix
+                            String scaled1xName = ImageTools.getScaledImageName(input, 1);
+                            theStream = ImageTools.createInputStream(scaled1xName);
+                        } catch (IOException e) {
+                            throw new IOException(e.getMessage() + " (original resource path: " + input + ")");
```

Comment:
I don't like this exception message, since it will now say something like
> yourImage@1x.jpeg was not found (original resource path: yourImage.jpeg)

That's confusing, because in all likelihood, I didn't specify that file name in my application code. What is the significance of mentioning `@1x.jpeg` in the exception message? We don't mention any of the other scale factors.

I'd rather just call `ImageTools.createInputStream(input)` again, and let the resulting exception bubble up.

We should _only_ mention the scale factor in the exception message if the scale factor was explicitly specified by the application developer.